.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Cypress Tests
```

Comment:
note: This [crash course](https://youtu.be/1oJQRlz1v94) helped me understand better Github actions. Jump to min ~12 to explain the YML syntax.